mibidata/mibi_image.py

```diff
@@ -219,6 +219,18 @@ def __getitem__(self, channels):
         """Equivalent to :meth:`~slice_data`."""
         return self.slice_data(channels)
 
+    def __repr__(self):
+        s = f"{type(self)}\n"
+        for key, val in self.metadata().items():
+            s += f'{key}: {val}\n'
+        return s
+
+    def __str__(self):
```

Comment:
`__str__` will default to be the same as `__repr__`, so you can omit this here.

Reply:
I tried that, but then I get \n in the string which looks ugly.

Reply:
Ah I hadn't noticed the distinction between the two. Got it.